release.sh

```diff
@@ -31,23 +31,27 @@ fi
 
 # Detect whether versions in code were updated.
 VER_FILE="version/version.go"
-TOML_TMPL_FILE="internal/pkg/scaffold/gopkgtoml.go"
```

Comment:
We also need to add the mention of `go_mod.go` to [this file](https://github.com/operator-framework/operator-sdk/blob/master/doc/dev/release.md#1-create-a-pr-for-release-version-and-changelogmd-updates).